launcher/src/components/UI/services-modal/SsvModal.vue

```diff
@@ -81,19 +87,37 @@ export default {
       selectedOperator: null,
       accepted: "",
       secretkey: null,
-      pubkey: null,
+      dataLoading: true,
     };
   },
 
   computed: {
     ...mapState(useNodeHeader, {
       runningServices: "runningServices",
       operators: "operators",
+      testOperatorData: "testOperatorData",
+    }),
+
+    ...mapWritableState(useNodeHeader, {
+      pubkey: "pubkey",
     }),
   },
   mounted() {
     this.getKeys();
   },
+  created() {
+    setTimeout(() => {
```

Comment:
what if the api call takes longer than 3 seconds? just do it as i stated in the previous comment and execute the functions in this component. and when you use `async await` you can just wait for the api result and make then all the checks